docs/ota-client-guide/modules/ROOT/nav.adoc

```diff
@@ -71,7 +71,7 @@ ifndef::env-github[:pageroot:]
 
 * xref:{pageroot}secure-software-updates.adoc[Secure your software repository]
 ** xref:{pageroot}install-garage-sign-deploy.adoc[Install the garage-deploy tool]
-** xref:{pageroot}rotating-signing-keys.adoc[Manage keys for software metadata]
+** xref:{pageroot}rotating-signing-keys.adoc[Rotate keys for root and targets metadata]
```

Comment:
"Root" and "Targets" should be capitalized here. We try to follow the [Uptane style guide](https://github.com/uptane/uptane-standard#style-guide) for consistency.

Reply:
Thanks for pointing that out! Should we also capitalize them in "root key" and "targets key"?

Reply:
Yes, I think so.

Reply:
Changed here and in all other instances.